### PR TITLE
feat: Open SusiSkills Activity by swapping from right to left

### LINKED ISSUE #1924: Swipe to go to susi-skills
**Describe the feature you'd like**

Swipe left in order to go to susi-skills and vice versa to come back. 
The user need not always go and touch the susi icon in order to go to the susi skills.
rather he can swipe left .

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #1924 

Changes: 
Added the code in ChatActivity.kt

Screenshots for the change: 
swipegif

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 1]
```diff
@@ -57,8 +60,10 @@ import java.util.Locale
  * The V in MVP
  * Created by chiragw15 on 9/7/17.
  */
-class ChatActivity : AppCompatActivity(), IChatView {
+class ChatActivity : AppCompatActivity(), IChatView, GestureDetector.OnGestureListener,
+        GestureDetector.OnDoubleTapListener {
 
+    private lateinit var mDetector: GestureDetectorCompat
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: We don't use hungarian notation

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 2]
```diff
@@ -57,8 +60,10 @@ import java.util.Locale
  * The V in MVP
  * Created by chiragw15 on 9/7/17.
  */
-class ChatActivity : AppCompatActivity(), IChatView {
+class ChatActivity : AppCompatActivity(), IChatView, GestureDetector.OnGestureListener,
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Don't implement the interfaces on the class, use anonymous classes and lambdas

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 3]
```diff
@@ -105,6 +111,68 @@ class ChatActivity : AppCompatActivity(), IChatView {
...
+        }
+    }
+
+    override fun onDown(event: MotionEvent): Boolean {
+
+        return true
+    }
+
+    override fun onFling(e1: MotionEvent?, e2: MotionEvent?, velocityX: Float, velocityY: Float): Boolean {
+        val X = e1!!.getX() - e2!!.getX()
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Using !! in kotlin is shooting yourself in the foot

**Comment by atm1504** [comment]
I, will correct the code accordingly and report soon.

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt (hunk 2 above)
Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Can you please explain how can I write it?

**Comment by atm1504** [comment]
@iamareebjamal I have changed few of the things that you have asked. But I am not getting "Don't implement the interfaces on the class, use anonymous classes and lambdas". Can you lease help me how I can do this. Thank you: )

**Comment by iamareebjamal** [comment]
Learn about anonymous classes and lambdas in Java. You'll understand

**Pull request closed without merging**